src/gql/queries/get-task-files.ts

```diff
@@ -3,6 +3,7 @@ import { gql } from "@apollo/client";
 export const GET_TASK_FILES = gql`
   query TaskFiles($id: String!, $execution: Int) {
     taskFiles(taskId: $id, execution: $execution) {
+      fileCount
```

Comment:
why not define a merge function for this query?

Reply:
As far as I know You can't define merge functions for top level fields in queries only sub fields with nested fields